src/cider/exec/template/IRCodegen.cpp

```diff
@@ -266,18 +266,16 @@ std::unique_ptr<CodegenColValues> CodeGenerator::codegenConstantExpr(
       return std::make_unique<TwoValueColValues>(
           constant_value[1],
           constant_value[2],
-          constant_expr->get_is_null()
-              ? llvm::ConstantInt::getTrue(cgen_state_->context_)
-              : llvm::ConstantInt::getFalse(cgen_state_->context_));
+          constant_expr->get_is_null() ? llvm::ConstantInt::getTrue(cgen_state_->context_)
+                                       : nullptr);
     case kARRAY:
       UNREACHABLE();
     default:
       CHECK_EQ(constant_value.size(), 1);
       return std::make_unique<FixedSizeColValues>(
           constant_value[0],
-          constant_expr->get_is_null()
-              ? llvm::ConstantInt::getTrue(cgen_state_->context_)
-              : llvm::ConstantInt::getFalse(cgen_state_->context_));
+          constant_expr->get_is_null() ? llvm::ConstantInt::getTrue(cgen_state_->context_)
+                                       : nullptr);
```

Comment:
seems old code path, need to change for nextgen?

Reply:
Yes, am on it, also get some uts failed.

Reply:
Changing this leads to some other errors for handling the nullptr.

Reply:
Reverted this part.